chirp/locale/check_parameters.py

```diff
@@ -10,7 +10,7 @@
 formatter = Formatter()
 
 for name, po in pos.iteritems():
-    print "Testing", name
+    print("Testing", name)
```

Comment:
I don't even know what this file was for back when it was relevant, but .. it's not anymore. What made you feel like you had to do this?

Let me just nuke this separately.

Reply:
Okay, I didn't write this so I didn't recognize it. I'll cherry pick this so it's not part of your driver PR.